consensus/service/src/tx_manager.rs

```diff
@@ -86,9 +79,9 @@ impl CacheEntry {
     }
 }
 
-/// A trait for representing the untrusted part of validation/combining. This is presented as a
-/// trait to make testing easier.
-pub trait UntrustedInterfaces: Clone {
+/// Transaction checks performed outside the enclave.
+#[automock]
```

Comment:
Should this be in a `#[cfg_attr(test, automock)]` (assuming it's only needed for test builds)?